testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -1,75 +1,69 @@
-# Copyright (c) 2015-2020 SUSE LLC
+# Copyright (c) 2015-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+@scc_credentials
 Feature: Be able to list available channels and enable them
   In order to use software channels
   As root user
   I want to be able to list available channels and enable them
 
-@scc_credentials
   Scenario: List available channels
     # Order matters here, refresh first
     When I refresh SCC
     And I execute mgr-sync "list channels -e" with user "admin" and password "admin"
-    Then I should get "[ ] SLES11-SP4-Pool for x86_64 SUSE Linux Enterprise Server 11 SP4 x86_64 [sles11-sp4-pool-x86_64]"
-    And I should get "    [ ] SLE11-SDK-SP4-Updates for x86_64 SUSE Linux Enterprise Software Development Kit 11 SP4 [sle11-sdk-sp4-updates-x86_64]"
+    Then I should get "[ ] SLES12-SP3-Pool for x86_64 SUSE Linux Enterprise Server 12 SP3 x86_64 [sles12-sp3-pool-x86_64]"
+    And I should get "    [ ] SLE-Live-Patching12-SP3-Pool for x86_64 SUSE Linux Enterprise Live Patching 12 SP3 x86_64 [sle-live-patching12-sp3-pool-x86_64]"
+    And I should get "sles12-sp3-debuginfo"
+    And I should get "sles12-sp3-installer"
 
-@scc_credentials
   Scenario: List available mandatory channels
     When I execute mgr-sync "list channels -e --no-optional"
-    Then I should get "[ ] SLES11-SP4-Pool for x86_64 SUSE Linux Enterprise Server 11 SP4 x86_64 [sles11-sp4-pool-x86_64]"
-    And I should get "    [ ] SLE11-SDK-SP4-Updates for x86_64 SUSE Linux Enterprise Software Development Kit 11 SP4 [sle11-sdk-sp4-updates-x86_64]"
-    And I shouldn't get "debuginfo"
-    And I shouldn't get "sles11-extras"
+    Then I should get "[ ] SLES12-SP3-Pool for x86_64 SUSE Linux Enterprise Server 12 SP3 x86_64 [sles12-sp3-pool-x86_64]"
+    And I should get "    [ ] SLE-Live-Patching12-SP3-Pool for x86_64 SUSE Linux Enterprise Live Patching 12 SP3 x86_64 [sle-live-patching12-sp3-pool-x86_64]"
+    And I shouldn't get "sles12-sp3-debuginfo"
+    And I shouldn't get "sles12-sp3-installer"
 
-@scc_credentials
 @susemanager
   Scenario: List products
     When I execute mgr-sync "list products"
-    Then I should get "[ ] SUSE Linux Enterprise Server 12 SP4 x86_64"
-    And I should get "[ ] SUSE Manager Proxy 4.0 x86_64"
+    Then I should get "[ ] SUSE Linux Enterprise Server 12 SP5 x86_64"
+    And I should get "[ ] SUSE Manager Proxy 4.1 x86_64"
 
-@scc_credentials
 @susemanager
   Scenario: List all products for SUSE Manager
     When I execute mgr-sync "list products --expand"
-    Then I should get "[ ] SUSE Linux Enterprise Server 12 SP4 x86_64"
-    And I should get "[ ] SUSE Manager Proxy 4.0 x86_64"
+    Then I should get "[ ] SUSE Linux Enterprise Server 12 SP5 x86_64"
+    And I should get "[ ] SUSE Manager Proxy 4.1 x86_64"
```

Comment:
```suggestion
    And I should get "[ ] SUSE Manager Proxy 4.3 x86_64"
```

Reply:
No, intentionally using old stuff that are meant to be killed, see my explanations

Let's reposync 4.3 separately.